Sources/MapboxNavigation/NavigationViewportDataSource.swift

```diff
@@ -345,7 +345,7 @@ public class NavigationViewportDataSource: ViewportDataSource {
             if options.followingCameraOptions.paddingUpdatesAllowed || followingMobileCamera.padding == nil {
                 followingMobileCamera.padding = UIEdgeInsets(top: followingMobileCameraAnchor.y,
                                                              left: viewportPadding.left,
-                                                             bottom: UIScreen.main.bounds.height - followingMobileCameraAnchor.y + 1.0,
+                                                             bottom: mapView.bounds.height - followingMobileCameraAnchor.y + 1.0,
```

Comment:
Do we need to apply the same fix for CarPlay on line 354?
```mainCarPlayScreen.bounds.height```
I'm wondering if a mapView can take only a part of a screen on CarPlay.